src/components/Grid/Grid.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { range } from "../../utils.js";
+import { NUM_OF_GUESSES_ALLOWED } from "../../constants.js";
+import { checkGuess } from "../../game-helpers.js";
+
+function PreviousGuesses({ guessHistory, correctAnswer }) {
+  const GuessRow = ({ numGuess }) => {
+    let checkedLetters;
+    if (guessHistory[numGuess]) {
+      const currGuess = guessHistory[numGuess];
+      checkedLetters = checkGuess(currGuess, correctAnswer);
+    }
```

Comment:
nit: JS cult people prefers `const` over `let` (which removes the unnecessary if branch) 

```suggestion
    const currGuess = guessHistory[numGuess];
    const checkedLetters = lastGuess ? checkGuess(currGuess, correctAnswer) : undefined;
```

Reply:
no i will not join your cult.

`lastGuess` doesn't exist, did you mean [Last Kiss](https://www.youtube.com/watch?v=idWma942CUI)? Or maybe [The Last Time](https://www.youtube.com/watch?v=pCH4QrSx2Jg)?